src/components/common/Notifications/index.tsx

```diff
@@ -2,21 +2,58 @@ import { ReactElement, SyntheticEvent, useCallback, useEffect } from 'react'
 import groupBy from 'lodash/groupBy'
 import { useAppDispatch, useAppSelector } from '@/store'
 import { closeNotification, Notification, selectNotifications } from '@/store/notificationsSlice'
-import { Alert, AlertColor, Snackbar, SnackbarCloseReason } from '@mui/material'
+import { Alert, AlertColor, Link, Snackbar, SnackbarCloseReason } from '@mui/material'
 import css from './styles.module.css'
+import NextLink from 'next/link'
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
+import { OVERVIEW_EVENTS } from '@/services/analytics/events/overview'
+import Track from '../Track'
 
 const toastStyle = { position: 'static', margin: 1 }
 
-const Toast = ({ message, severity, onClose }: { message: string; severity: AlertColor; onClose: () => void }) => {
+export const NotificationLink = ({
+  link,
+  onClick,
+}: {
+  link: Notification['link']
+  onClick: (_: Event | SyntheticEvent) => void
+}): ReactElement | null => {
+  if (!link) {
+    return null
+  }
+
+  return (
+    <Track {...OVERVIEW_EVENTS.NOTIFICATION_INTERACTION} label={link.title}>
+      <NextLink onClick={onClick} href={link.pathname} passHref>
+        <Link className={css.link} sx={{ mt: 1 }}>
```

Comment:
I would suggest adding the margin to the css class instead so its all in one place.